.github/workflows/pkgdown_dev.yaml

```
@@ -40,7 +40,7 @@ jobs:
       - name: Build site
         env:
           DELPHI_EPIDATA_KEY: ${{ secrets.SECRET_EPIPROCESS_GHACTIONS_DELPHI_EPIDATA_KEY }}
-        run: pkgdown::build_site_github_pages(new_process = FALSE, install = FALSE)
+        run: pkgdown::build_site_github_pages(new_process = FALSE, install = FALSE, override=list(PKGDOWN_DEV_MODE="devel"))
```

issue (nonblocking): For maintenance, prefer to have both `dev` and `main` behavior in the same workflow. We should be able to set `PKGDOWN_DEV_MODE="devel"` or `PKGDOWN_DEV_MODE="release"` conditionally on the push/pull branch.

This is untested:

```suggestion
        run: |
            if (startsWith("${{ GITHUB_EVENT_NAME }}", "pull_request")) {
                mode <- ifelse("${{ GITHUB_BASE_REF }}" == "main", "release", "devel")
            } else {
                mode <- ifelse("${{ GITHUB_REF_NAME }}" == "main", "release", "devel")
            }
            pkgdown::build_site_github_pages(new_process = FALSE, install = FALSE, override=list(PKGDOWN_DEV_MODE=mode))
```

[GH actions var reference](https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables)

---

Not sure why `if (startsWith("${{ GITHUB_EVENT_NAME }}", "pull_request"))` exists, seems to be the same in either case.

---

If it's a pull request, you have to get the branch name from a different variable (GITHUB_BASE_REF vs GITHUB_REF_NAME)

---

There might be a better way to do this conditional logic, too. This is just my initial thought on how to approach it.

---

gotcha, didn't read closely enough. Seems like it should work.